challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map(&:capitalize)
 end
 
 def fetch_the_dog(input)
-  #implement your solution here
+  input.find_all { | word | word == "dog" }
 end
 
 def no_dogs_allowed(input)
-  #implement your solution here
+  input.reject { | word | word == "dog" }
 end
 
 def count_the_animals(input)
-  #implement your solution here
+  input.count
 end
 
 def fetch_the_first_two(input)
-  #implement your solution here
+  input.take(2)
 end
 
 def fetch_CD_animals(input)
-  #implement your solution here
+  input.grep(/\A[cCdD]/)
```

Comment:
🏆 .  Don't bother fixing it, but there should be a way to stick an `i` in there somewhere to make your grepping case-insensitive.